### ISSUE TITLE
Disabling multiple cops in the same namespace not possible

### ISSUE THREAD

**Issue by NobodysNightmare** [rubocop-hq/rubocop#7789]
Previous versions of rubocop allowed me to disable all cops of a namespace (`Metrics`, `Style` etc) like this

#### .rubocop.yml
```yaml
Style:
  enabled: false
```

#### rubocop.rb
```ruby
foo 'bar'
```

#### Executing rubocop
```bash
rubocop rubocop.rb
```

## Expected behavior

Rubocop should not detect any offense, since the only violation in that file is `Style/FrozenStringLiteralComment`.

## Actual behavior

```
$ rubocop rubocop.rb 
[ --- snipped out hints about pending cops --- ]
Inspecting 1 file
C

Offenses:

rubocop.rb:1:1: C: Style/FrozenStringLiteralComment: Missing frozen string literal comment.
foo 'bar'
^

1 file inspected, 1 offense detected
```

## RuboCop version

```
$ rubocop -V
0.80.1 (using Parser 2.7.0.2, running on ruby 2.6.4 x86_64-linux)
```

## Remarks

I tried to figure out, whether this feature was intentionally removed, but I feel like it should still be supported, because the above configuration does not raise an error, while the following configuration (very expectably) raises an error:

```yaml
Styl:
  enabled: false
```

```
$ rubocop rubocop.rb 
Error: unrecognized cop Styl found in .rubocop.yml
```

Since rubocop doesn't complain about `Style` being an unrecognized cop, I assume that it is still supposed to filter all `Style` cops.

Thanks a lot for your time and effort maintaining this gem. We love using it every day :heart:

**Comment by koic** [comment]
`enabled: false` would be `Enabled: false` correctly.

```diff
 Style:
-  enabled: false
+  Enabled: false
```

**Comment by NobodysNightmare** [comment]
Okay, that sends me back to the drawing board with my bug report.

I tried to simplify the reproduction and made that mistake :(

Will close until I can figure it out.
